### PR TITLE
Nerfs Spears and Pikes

### PR DESCRIPTION
**Pull request by Kaedwuff**
These weapons had a very high damage per hardness value (75%), causing situations where materials with high hardness such as plasteel (hardness 80) and diamond (hardness 100) could create stupidly unbalanced and powerful weapons that are capable of dealing base damages of 60 damage (for plasteel) or 75 (for diamond) per hit.  

For comparison, an esword, one of the stronger antag items, has a base damage of 30, and the energy glaive, something generally only found by adminnery, is 40. 

Keep in mind also the average character has a maximum of 200 hp, and going below 100 is enough to crit someone enough to pass out and need medical attention or they die. 

With these new values even a diamond spear has been brought down to a much more manageable 35 base when two handed.

Also, gold and uranium are much softer, but also heavier, and better for using on thrown weapons than melee.

### CODE REVIEW

**Comment by Kaedwuff** [comment]
!review

**Comment by alsoandanswer** [comment]
Seems good to me, although you should probably re-word that changelog to be less condescending, 
lest ye get lynched at the forums

you *are* nerfing something OP that players have been exploiting for a while.

**Comment by Kaedwuff** [comment]
I thought it was more humorous than condescending, but as you wish.  As for the damage values, I sort of included them in the comments.  I'll be more clear though.

File: code/game/objects/items/weapons/material/twohanded.dm [hunk 1]
```diff
@@ -540,8 +540,8 @@
 	name = "halberd"
 	desc = "A sharp axe mounted on the top of a long spear."
 	force = 10
-	unwielded_force_divisor = 0.4
-	force_divisor = 0.6
+	unwielded_force_divisor = 0.2
```
Comment (Alberyk) [inline code/game/objects/items/weapons/material/twohanded.dm]: You can't make halberds, you can only get them via the merchant and adminbus. No reason to nerf them.

Comment (Kaedwuff) [reply code/game/objects/items/weapons/material/twohanded.dm]: Merchant/adminbus or not, those values still make a plasteel/diamond halberd stronger than an energy glaive.  28 per hit damage while dual wielded is still plenty to be going along with for a plasteel halberd (or 35 for a diamond one).

Comment (Alberyk) [reply code/game/objects/items/weapons/material/twohanded.dm]: You can't make halberds with any other material, they are always steel.

Comment (Kaedwuff) [reply code/game/objects/items/weapons/material/twohanded.dm]: Ah, that makes things different.  Hokay.

Comment (Alberyk) [reply code/game/objects/items/weapons/material/twohanded.dm]: Also, not sure how this nerfs pikes if it is touching halberds.

Comment (Kaedwuff) [reply code/game/objects/items/weapons/material/twohanded.dm]: Halberds are a child of pikes, so it doesn't affect pikes themselves at all.

Comment (Alberyk) [reply code/game/objects/items/weapons/material/twohanded.dm]: You can only make the regular pikes, not halberds.

**Comment by alsoandanswer** [comment]
Should probably check if any child weapons from pikes get screwed up

File: code/game/objects/items/weapons/material/twohanded.dm [hunk 2]
```diff
@@ -218,9 +218,9 @@
 	force = 10
 	w_class = 4.0
 	slot_flags = SLOT_BACK
-	force_wielded = 0.75           // 22 when wielded with hardness 15 (glass)
-	unwielded_force_divisor = 0.65 // 14 when unwielded based on above
-	thrown_force_divisor = 1.5 // 20 when thrown with weight 15 (glass)
+	force_wielded = 0.35           // 22 when wielded with hardness 15 (glass)
```
Comment (Lohikar) [inline code/game/objects/items/weapons/material/twohanded.dm]: This is the wrong var, note. This should be setting `force_divisor`.

**Comment by Kaedwuff** [comment]
> Should probably check if any child weapons from pikes get screwed up

The only other child of pikes is pitchforks and it has its own values I haven't messed with.

**Comment by VTCobaltblood** [comment]
> As for the damage values, I sort of included them in the comments. 

I mean including them in code. Code had this before:

```
// 22 when wielded with hardness 15 (glass)
// 14 when unwielded based on above
// 20 when thrown with weight 15 (glass)
```

Can you update these for new material values?

**Comment by Kaedwuff** [comment]
Those notations aren't you the code anymore, I removed them.

**Comment by VTCobaltblood** [comment]
...Yes, I am asking you to bring them back with new values.

**Comment by Kaedwuff** [comment]
> ...Yes, I am asking you to bring them back with new values.

There.

**Comment by Kaedwuff** [comment]
Miscalculated some math - thrown damage is based on weight, not hardness. 

This should bring it in line with the rest, with one special thing: Being the heaviest metal, gold has the special quality of being very strong when thrown, but almost useless as a melee weapon.  This should create some interesting risk/reward options.

**Comment by Kaedwuff** [comment]
Is there anything else that needs to be fixed here?

**Comment by Kaedwuff** [comment]
!review

**Comment by Kaedwuff** [comment]
This continues to sit here unfixed and unchecked. Is there something else that needs to be changed?

**Pull request merged**